src/modules/login/index.tsx

```diff
@@ -25,27 +69,142 @@ export default function LoginScreen() {
 
   return (
     <View style={styles.container}>
-      <Text style={styles.title}>LOGIN</Text>
-      <Button
-        onPress={openTabs}
-        title="Go to application"
-        color="#841584"
-      />
+      <View style={styles.header}>
+        <Image
+            source={require('../../../assets/images/logo.png')}
+        />
+      </View>
+      <View style={styles.footer}>
+        <View>
+          <TextInput
+              style={[styles.input, data.isFocusedText && styles.isFocused]}
+              placeholder="Username"
+              onChangeText={(val) => TextInputChange(val)}
+              onFocus={updateIsFocusedText}
+              onBlur={updateIsFocusedText}
+          />
+        </View>
+        <View>
+          <TextInput
+              style={[styles.input, data.isFocusedPassword && styles.isFocused]}
+              placeholder="Password"
+              secureTextEntry={data.secureTextEntry}
+              onChangeText={(val) => handlePasswordChange(val)}
+              onFocus={updateIsFocusedPassword}
+              onBlur={updateIsFocusedPassword}
+          />
+          <TouchableOpacity
+              style={styles.touchableOpacity}
+              onPress={updateSecureTextEntry}
+          >
+            {data.secureTextEntry ?
+                <Feather
+                    style={styles.icon}
+                    name='eye-off'
+                    color='#5F768A'
+                    size={20}
+                />
+                :
+                <Feather
+                    style={styles.icon}
+                    name='eye'
+                    color='#5F768A'
+                    size={20}
+                />
+            }
```

Comment:
Looks like the difference between them is just a name. Would it make more sense to use it next way:

```
<Feather
                    style={styles.icon}
                    name={data.secureTextEntry ? 'eye-off' : 'eye'}
                    color='#5F768A'
                    size={20}
                />
```